glean-core/android/src/test/java/mozilla/telemetry/glean/private/AccumulationsBeforeGleanInitTest.kt

```diff
@@ -33,6 +34,8 @@ class AccumulationsBeforeGleanInitTest {
     @Before
     fun cleanup() {
         Glean.testDestroyGleanHandle()
+        @Suppress("EXPERIMENTAL_API_USAGE")
+        Dispatchers.API.setTaskQueueing(true)
```

Comment:
We may also want to clear the queue to be extra sure...

Reply:
Yes, but blindly clearing breaks other tests. We should take that in a follow-up, see the filed bug for it.

Reply:
"Fun"